examples/tornado/app.py

```diff
@@ -9,9 +9,9 @@
 
 
 @app.middleware  # or app.use(log_request)
-def log_request(logger, body, next):
+def log_request(logger, body, next_):
```

Comment:
just a minor improvement to eliminate warnings by IDE. See also: https://github.com/slackapi/bolt-python/pull/394